test/integration.test.ts

```diff
@@ -600,18 +601,62 @@ const testCases: {
       expect(tableHeads).toContain('Exports')
       expect(tableHeads).toContain('File')
       expect(tableHeads).toContain('Size')
+      const [exportsIndex, fileIndex, sizeIndex] = [
+        tableHeads.indexOf(pc.underline('Exports')),
+        tableHeads.indexOf(pc.underline('File')),
+        tableHeads.indexOf(pc.underline('Size')),
+      ]
 
+      // meet trouble to get index
       expect(cliLine).toContain('cli (bin)')
+      expect(cliLine.indexOf('cli (bin)')).toEqual(exportsIndex)
       expect(cliLine).toContain('dist/cli.js')
+      expect(cliLine.indexOf(pc.bold('dist/cli.js'))).toEqual(fileIndex)
 
       expect(indexLine).toContain('.')
+      expect(indexLine.indexOf('.')).toEqual(exportsIndex)
       expect(indexLine).toContain('dist/index.js')
+      expect(indexLine.indexOf(pc.bold('dist/index.js'))).toEqual(fileIndex)
 
       expect(indexReactServerLine).toContain('. (react-server)')
+      expect(indexReactServerLine.indexOf('. (react-server)')).toEqual(
+        exportsIndex,
+      )
       expect(indexReactServerLine).toContain('dist/index.react-server.js')
+      expect(
+        indexReactServerLine.indexOf(pc.bold('dist/index.react-server.js')),
+      ).toEqual(fileIndex)
 
       expect(fooLine).toContain('./foo')
+      expect(fooLine.indexOf('./foo')).toEqual(exportsIndex)
       expect(fooLine).toContain('dist/foo.js')
+      expect(fooLine.indexOf(pc.bold('dist/foo.js'))).toEqual(fileIndex)
+    },
+  },
+  {
+    name: 'output-short',
+    args: ['./o.js', '-f', 'cjs', '-o', './o.cjs', '--no-clean'],
```

Comment:
This would be test in `cli.test.ts`, for this integration one we need to test a smaller app without `cli` and maybe only `.` export.

And add another test in cli for the output

Reply:
got it.